src/value_generators/test/DocumentGeneratorTestCases.yml

```diff
@@ -527,8 +539,8 @@ Tests:
     ThenReturns:
     # The $date notation is extended json
     # (see https://docs.mongodb.com/manual/reference/mongodb-extended-json/#bson.Date).
-    - { "date" : { "$date" : { "$numberLong" : "1778468968160" } } }
-    - { "date" : { "$date" : { "$numberLong" : "1871664110292" } } }
+    - { "date" : { "$date" : { "$numberLong" : "4185670857742" } } }
+    - { "date" : { "$date" : { "$numberLong" : "5397029274960" } } }
```

Comment:
max date changed in the last commit which has a knock on effect on the dates generated.